GeoJSON/Geometry.swift

```diff
@@ -129,6 +129,8 @@ open class Position: NSObject {
         
         latitude = coordinates.count > 1 ? coordinates[1] : 0.0
         longitude = coordinates.count > 0 ? coordinates[0] : 0.0
+        
+        longitude = longitude.mappedLongitude
```

Comment:
Happy with this change, but worried about the function below this switching the order!! Comments @simonmitchell ? :)

Reply:
Erghhhhh yeah not ideal :/ problem is some of our GeoJSON providers do seem to provide coordinates in the incorrect order (can't remember exactly the ones that do this) 😟 Maybe we need to make the `init` method take the `coordinateOrder`? Not sure what knockoff effects that will have though...

Reply:
Well this **needs** to be handled at the reading of the dictionary level, storing the longitude value in `latitude` when it really means `longitude` is not nice at all!! 
This fix would then be fine.

Otherwise I'd propose we need to change these property names against `Position`.

Reply:
I think move it to the reading or the init level then as suggested :) Agreed this is not pretty in the slightest. I'm just hoping there's no good reason it was done like this (Easily serialising back to JSON perhaps in the correct coordinate order for example)